### ISSUE TITLE
Can gun be used with Deno

### ISSUE THREAD

**Issue by cognociente** [amark/gun#982]
New node alternative from Ryan Dahl.

**Comment by backspaces** [comment]
I agree, we've moved to deno simply because it uses the browser's javascript and es6 imports with none of node's odd functions. It also has easy to use edge services.
